### ISSUE TITLE
[Item] Void Star Talisman - Warlock Class Trinket

### ISSUE THREAD

**Issue by MorbidMike** [FeenixServerProject/Archangel_2.4.3_Bugtracker#2251]
Good day!

I just acquired the Warlock class trinket from Solarian, Void Star Talisman. It is supposed to give 48 Spell Damage and 130 resistances to my pet. 

The Spell Damage part is working correctly, however, the pet resistances are not. My pet's window does not show the added resistances, and they do not seem to be added either (spells hit my pet for the same amount of damage before and after equipping VST), which leads me to conclude that the Void Star Talisman's pet resistance bonus is not functioning properly.

In short: Voidstar Talisman pet resistance are not added to my pet's resistances.

**Comment by Pawx** [comment]
The trinket still does not work. Can you please have a look into this? 

When equiped, you only get the spell power bonus it gives and none of the pet resistances.

**Comment by devidao** [comment]
Fixed, coming soon on PTR.

**Comment by Nzzoth** [comment]
Fix denied on PTR.
